src/Routes/ImageManager/Images.js

```diff
@@ -22,14 +23,20 @@ const UpdateImageWizard = React.lazy(() =>
   )
 );
 
-const Images = () => {
-  const history = useHistory();
-  const { pathname, search } = useLocation();
+const Images = ({ historyProp, locationProp, navigateProp }) => {
+  const history = historyProp
+    ? historyProp()
+    : navigateProp
+    ? useNavigate()
+    : useHistory();
+  const navigate = navigateProp ? navigateProp() : useNavigate();
```

Comment:
We should return null when navigate isn't available
```suggestion
  const navigate = navigateProp ? navigateProp() : useNavigate ? useNavigate() : null;
```